code/modules/mob/holder.dm

```diff
@@ -278,7 +275,13 @@ var/list/holder_mob_icon_cache = list()
 	var/list/generate_for_slots = list(slot_l_hand_str, slot_r_hand_str, slot_back_str)
 	slot_flags = SLOT_BACK
 
-/obj/item/weapon/holder/human/proc/sync(var/mob/living/M)
```

Comment:
i decided removing this proc from nonhuman holders was a bad idea, ive reinstated it, though in a very light form. just to keep clutter out of the get_scooped proc